packages/server/src/rpc/codes.ts

```diff
@@ -34,6 +34,7 @@ export const TRPC_ERROR_CODES_BY_KEY = {
   UNPROCESSABLE_CONTENT: -32022, // 422
   TOO_MANY_REQUESTS: -32029, // 429
   CLIENT_CLOSED_REQUEST: -32099, // 499
+  I_AM_A_TEAPOT: -32018, // 418
```

Comment:
I'd still sort it nonetheless :P should go on line 34

Reply:
good catch

Reply:
You missed `www/docs/server/error-handling.md` :(

Reply:
Lmao ok, I'll do that one too